php/autocomplete.php

```diff
@@ -52,22 +53,41 @@ function trim_query($query) {
     // >4 chars: match on name or city
 
     $ext = "";
-    $sort_order = "iata IS NULL,icao IS NULL,city,name";
+    $sort_order = "iata IS NULL, icao IS NULL, city, name";
     if (strlen($searchText) <= 3) {
         $ext = "iata != '' AND iata != :code AND";
     } elseif ($multi) {
         // Exclude private airports from multisearch
         $ext = "icao != '' AND";
     }
-    $sql = "SELECT 2 as sort_col,apid,name,city,country,iata,icao,x,y,timezone,dst FROM airports WHERE $ext (city LIKE :name";
+
+    $sql = <<<SQL
+        SELECT 2 as sort_col, apid, name, city, country, iata, icao, x, y, timezone, dst
+        FROM airports
+        WHERE $ext (city LIKE :name
+    SQL;
 
     switch (strlen($searchText)) {
         case 3: // IATA
-            $sql = "SELECT 1 as sort_col,apid,name,city,country,iata,icao,x,y,timezone,dst FROM airports WHERE iata=:code UNION ($sql)) ORDER BY sort_col,$sort_order LIMIT $limit";
+            $sql = <<<SQL
+                SELECT 1 as sort_col, apid, name, city, country, iata, icao, x, y, timezone, dst
+                FROM airports
+                WHERE iata=:code
+                UNION ($sql))
+                ORDER BY sort_col,$sort_order
+                LIMIT $limit
+            SQL;
             break;
 
         case 4: // ICAO
-            $sql = "SELECT 1 as sort_col,apid,name,city,country,iata,icao,x,y,timezone,dst FROM airports WHERE icao=:code UNION ($sql)) ORDER BY sort_col,$sort_order LIMIT $limit";
+            $sql = <<<SQL
+            SELECT 1 as sort_col, apid ,name ,city ,country ,iata ,icao ,x , y, timezone, dst
```

Comment:
Fixed some spaces and commas.

```suggestion
            SELECT 1 as sort_col, apid, name, city, country, iata, icao, x, y, timezone, dst
```